aix/SPECS/wazuh-agent-aix.spec

```diff
@@ -302,7 +314,7 @@ rm -fr %{buildroot}
 - More info: https://documentation.wazuh.com/current/release-notes/
 * Wed Oct 06 2021 support <info@wazuh.com> - 4.2.3
 - More info: https://documentation.wazuh.com/current/release-notes/
-* Wed Oct 06 2021 support <info@wazuh.com> - 4.2.2
+* Tue Sep 28 2021 support <info@wazuh.com> - 4.2.2
```

Comment:
Fix 4.1.5, not Monday

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/1677/commits/42feb16ca391c17786ee0ba99a222c2e06c4ec24